src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -38,25 +45,32 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
     @observable placement: 'left' | 'right' = 'right';
     @observable chartType: ChartType
 
-    @computed get fileName() {
-        return this.props.chartMeta.clinicalAttribute.displayName.replace(/[ \t]/g, '_');
+    @computed
+    get fileName() {
+        return this.props.chartMeta.displayName.replace(/[ \t]/g, '_');
```

Comment:
why not just all spaces?

Reply:
@alisman I think we just use underscore over space in the file name as a convention, but we could consolidate the rules with other pages. Any logic we have in result's page?